README.md

```
@@ -8,10 +8,10 @@ Components
 
 ----------
 Majestik is composed of two core components:
-2. Compiler: This component translates the AST produced by Magik-Tools' Magik-Squid into JVM bytecode using the Class-File API.
+1. Compiler: This component translates the AST produced by Magik-Tools' Magik-Squid into JVM bytecode using the Class-File API.
```

Markdownlint (0.37.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)